### DISCUSSION TITLE
using expect in playwright tests while running in artillery

### DISCUSSION THREAD

**Discussion by jessehattabaugh** [artilleryio/artillery#1439] [Artillery]
I am using `artillery-engine-playwright` and I have the example working but when I try to add an assertion like so:

```js
const invalidCredentials = page.locator('text=Invalid Credentials"');
await expect(invalidCredentials).toBeVisible();
```

I get the error `locator._expect: expression: expected string, got object`. The artillery scenario runs, but the assertion fails.

Am I doing something wrong, or are assertions not supported by `artillery-engine-playwright`?

**Comment by jessehattabaugh** [comment]
I'm importing `expect` like so:

```js
const { expect } = require('@playwright/test');
```

**Reply by hassy** [reply]
hi @jessehattabaugh 👋 just to confirm - does the same scenario run as expected when run directly (vs through Artillery)?

**Reply by jessehattabaugh** [reply]
no, but that's because I don't use `@playwright/test`'s `test()` function.
